modules/ROOT/pages/index.adoc

```diff
@@ -53,6 +53,8 @@ See xref:logs.adoc[Logs].
 
 Limits on data storage and retention are determined by your subscription level (Base or Titanium). All customers receive Anypoint Monitoring at the Base subscription level.
 
+For Base, Gold, and Platinum subscriptions, data is stored in US East (N. Virginia).
```

Comment:
"Base" is more a description than an actual subscription license. Remove Base and this looks great!

Reply:
```suggestion
For Gold and Platinum subscriptions, data is stored in US East (N. Virginia).
```